pkg/cli/advisory_export.go

```diff
@@ -67,6 +77,7 @@ func AdvisoryExport() *cobra.Command {
 				defer outputFile.Close()
 			}
 
+			fmt.Println(export)
```

Comment:
What are you thinking for this line? I think we're already outputting the exported data using the user's `p.outputLocation` selection, is that right?

Reply:
Whoops, that should have been removed.